.github/workflows/long_faucet_chain_test.yml

```diff
@@ -0,0 +1,37 @@
+name: Long Faucet chain test
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+  CARGO_NET_RETRY: 10
+  LINERA_STORAGE_SERVICE: 127.0.0.1:1235
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest-16-cores
```

Comment:
Is that needed for this test?
I think the parallelism is not exploited in a significant way by this test.